libs/sdk-backend-spi/src/workspace/styling/theme.ts

```diff
@@ -92,6 +110,11 @@ export interface IThemePalette {
      * Used to express info or progress
      */
     info?: IThemeColorFamily;
+
+    /**
+     * Complementary palette
+     */
```

Comment:
this doc should be expanded to describe what is the complementary palette used for. what can be achieved using it.

Reply:
Makes sense. Take a look at force-pushed changes now.